pkg/heartbeat/language.go

```diff
@@ -1653,19 +1742,29 @@ const (
 	languageFactorStr                      = "Factor"
 	languageFancyStr                       = "Fancy"
 	languageFantomStr                      = "Fantom"
+	languageFaustStr                       = "Faust"
 	languageFelixStr                       = "Felix"
 	languageFennelStr                      = "Fennel"
+	languageFIGletFontStr                  = "FIGlet Font"
+	languageFilebenchWMLStr                = "Filebench WML"
+	languageFilterscriptStr                = "Filterscript"
 	languageFlatlineStr                    = "Flatline"
 	languageFloScriptStr                   = "FloScript"
-	languageForthStr                       = "Forth"
 	languageFishStr                        = "Fish"
+	languageFLUXStr                        = "FLUX"
 	languageFontStr                        = "Font"
+	languageFormattedStr                   = "Formatted"
+	languageForthStr                       = "Forth"
 	languageFortranStr                     = "Fortran"
 	languageFortranFixedStr                = "FortranFixed"
+	languageFortranFreeFormStr             = "Fortran Free Form"
 	languageFSharpStr                      = "F#"
 	languageFoxProStr                      = "FoxPro"
 	languageFreefemStr                     = "Freefem"
-	languageFStarStr                       = "FStar"
```

Comment:
`FStar` is used in chroma. As it seems to be valid alternative spelling for `F*`, let's add it to the general aliases (https://github.com/wakatime/wakatime-cli/blob/develop/pkg/heartbeat/language_test.go#L738), as well as to `StringChroma` (https://github.com/wakatime/wakatime-cli/blob/develop/pkg/heartbeat/language_test.go#L939)